components/Login.tsx

```diff
@@ -31,22 +62,22 @@ function Login() {
         <View style={[styles.container, loginStyles.container]}>
           <Text style={[loginStyles.heading]}>Login</Text>
           <TextInput
-            value={''}
-            onChangeText={() => {}}
-            placeholder={'Username'}
+            value={email}
+            onChangeText={setEmail}
+            placeholder={'E-mail'}
             style={[loginStyles.input]}
           />
           <TextInput
-            value={''}
-            onChangeText={() => {}}
-            placeholder={'Password'}
+            value={password}
+            onChangeText={setPassword}
+            placeholder={'PIN'}
             secureTextEntry={true}
             style={loginStyles.input}
           />
           <TouchableOpacity
             activeOpacity={touchableStyles}
             style={[loginStyles.button]}
-            onPress={() => {}}
+            onPress={noGoogleLogin}
```

Comment:
Could we use a different name? Like email login or something?

Reply:
yes

Reply:
changed to emailLogin